src/arbitraries/MappedArbitrary.ts

```diff
@@ -32,8 +39,17 @@ export class MappedArbitrary<A, B> extends Arbitrary<B> {
   }
 
   canGenerate(pick: FluentPick<B>) {
-    /* && pick.value === this.f(pick.original) */
-    return this.baseArbitrary.canGenerate({value: pick.original, original: pick.original})
+    const inverseValues = !(this.config && this.config.inverseFunction) ? [pick.original] :
```

Comment:
I think you can simplify `this.config && this.config.inverseFunction` to `this.config?.inverseFunction`.

Reply:
I know the problem is that for some reason that I could not understand I get a syntax error. I tried to research to find why it is giving me an error but I cannot find the reason for the problem.

```
        const inverseValues = !(this.config?.inverseFunction) ? [pick.original] :
                                            ^

SyntaxError: Unexpected token '.'
```

Reply:
I already figured it out. The problem was related with some problems with node and my ubuntu that was always resetting my node version to one that did not support using `?.`. Btw already made the change.